tasks/components/watchdog/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+# Check if the variable is not blank
+if [[ ! -z "$CONFIG_DS_WATCHDOG_DEVICE" ]]; then
```

Comment:
Shouldn't this trigger on `DS_WATCHDOG`?

Otherwise, both `DS_WATCHDOG_DEVICE` and `DS_WATCHDOG_TIMEOUT` have defaults. If you specifically want to check that device is set, we should also check that the timeout is set.

Reply:
This should be fixed now. I added the default later thinking its going to be less error prone, but if we're requiring the user to set a device anyway this check can just be removed.